### DISCUSSION TITLE
Lists and Stars could be a tab + other ideas

### DISCUSSION THREAD

**Discussion by mishmanners** [community/community#8231] [Other Feature Feedback, Questions, & Ideas]
Would be great if the lists/stars come up as a tab under our profile.

From the looks of the URL: www.github.com/mishmanners/?tab=stars (this will show tabs and stars) this is meant to be a tab that would show up under profile as a tab alongside Overview, Repos, etc.
![Tabs](https://user-images.githubusercontent.com/36594527/143958848-7b0122a0-d3a7-48c4-af5f-bad201ce7776.JPG)

Would be also great to have lists come up under our profile drop down under discussions, stars, gists, etc.
![Drop down](https://user-images.githubusercontent.com/36594527/143958863-659bff14-b553-4c08-87aa-c2ec1ac33ede.JPG)

Would be also awesome be able to share and collaborate on lists; depends on what the goal of "lists" is however.

**Comment by brianlovin** [comment]
We've added the `Stars` tab back to profile navigation!

And +1 to collaborative lists — the team has been thinking about how this might work, but no plans to build it quite yet 😊
